src/modules/home/components/LoggedInHome/LoggedInHome.tsx

```diff
@@ -271,7 +271,8 @@ const LoggedInHome: FC<DefaultSeoInfo> = ({ url }) => {
 						</p>
 					</section>
 
-					{(active?.items || []).length > 0 && (
+					{(active?.items.filter((visit) => !visit.id.includes('permanent-id')) || [])
```

Comment:
i would store these arrays as computed values
so you can also use them for the actual generation of the visit space cards

Because now you render all cards if at least one card does not contain a permanent id

But this logic would be better: render only cards that do not have permanent ids, and render nothing if there are none

eg:
```typescript
const actualActiveVisitRequests = active?.items.filter((visit) => !visit.id.includes('permanent-id')) || [];
const actualFutureVisitRequests = future?.items.filter((visit) => !visit.id.includes('permanent-id')) || [];
const actualPendingVisitRequests = pending?.items.filter((visit) => !visit.id.includes('permanent-id')) || [];

{actualActiveVisitRequests.length > 0 && (<section .... actualActiveVisitRequests.map(...) ...)
{actualFutureVisitRequests .length > 0 && (<section .... actualFutureVisitRequests.map(...) ...)
{actualPendingVisitRequests .length > 0 && (<section .... actualPendingVisitRequests.map(...) ...)
```